models/pipeline.js

```diff
@@ -11,6 +11,16 @@ const Parameters = require('../config/parameters');
 const mutate = require('../lib/mutate');
 
 const STATES = ['ACTIVE', 'INACTIVE'];
+const BADGE = Joi.object({
```

Comment:
```suggestion
const SONAR = Joi.object({
```

Reply:
Oh, this is not a typo. This is written for extendibility, so we might be able to add something like in the future.

```
{
    sonar: BADGE,
    saucelabs: BADGE
}
```